src/vsr/replica.zig

```diff
@@ -5523,7 +5523,17 @@ pub fn ReplicaType(
 
             // This check must precede any send_eviction_message_to_client(), since only the primary
             // should send evictions.
-            if (self.ignore_request_message_backup(message)) return true;
+            if (self.backup()) {
```

Comment:
Thinking out loud:

When a cluster running this code is combined with a client running the old code, the old client still works since even though its `register` won't be forwarded, it is round-robin-ing its requests, so it will stumble on the primary eventually. And also it has a 30s ping broadcast upper bound.

Reply:
Ah right, I got it backwards which one we forwarded!